Develappers.BillomatNet/ArticleService.cs

```diff
@@ -142,11 +142,11 @@ public async Task<Article> CreateAsync(Article model, CancellationToken token =
         {
             if (model == null)
             {
-                throw new ArgumentException();
+                throw new ArgumentException("article or a value of the article is null", nameof(model));
```

Comment:
" or a value of the article" prüfst du an der Stelle doch gar nicht. Bitte anpassen in z.B. "article is null"